### PR TITLE
chore(amplify_authenticator): validated country information

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Fixed a few country objects in the source used for code gen.
* Added Bonaire, Sint Eustatius and Saba
* Changed Swaziland to Eswatini
* Added Kosovo
* Added United States Minor Outlying Islands
* Removed Netherlands Antilles ([no longer exists](https://2009-2017.state.gov/r/pa/ei/bgn/22528.htm))
* [Corrected dial codes](https://countrycode.org/) for Cayman Islands, Kazakhstan, Pitcairn

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_authenticator/lib/src/l10n/country_resolver.dart [hunk 1]
```diff
@@ -20,1722 +22,1492 @@ import 'package:flutter/material.dart';
 enum CountryResolverKey {
   selectDialCode,
   noDialCodeSearchResults,
-
```
Comment (dnys1) [inline packages/amplify_authenticator/lib/src/l10n/country_resolver.dart:23]: Please run `flutter format .` from the authenticator package.

File: packages/amplify_authenticator/lib/src/l10n/country_resolver.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: constant_identifier_names, non_constant_identifier_names
```
Comment (dnys1) [inline packages/amplify_authenticator/lib/src/l10n/country_resolver.dart:1]: Why is this needed?

File: packages/amplify_authenticator/lib/src/utils/country_code.dart [hunk 3]
```diff
@@ -25,246 +26,250 @@ class Country {
 
 /// List of country information
 const countryCodes = [
-  Country('US', '1', CountryResolverKey.us$),
```
Comment (dnys1) [inline packages/amplify_authenticator/lib/src/utils/country_code.dart:28]: Please address this for the countries list

File: packages/amplify_authenticator/tool/countries.dart [hunk 4]
```diff
@@ -120,23 +119,26 @@ var countries = [
   {'name': 'Japan', 'dial_code': '+81', 'code': 'JP'},
   {'name': 'Jersey', 'dial_code': '+44', 'code': 'JE'},
   {'name': 'Jordan', 'dial_code': '+962', 'code': 'JO'},
-  {'name': 'Kazakhstan', 'dial_code': '+77', 'code': 'KZ'},
+  {'name': 'Kazakhstan', 'dial_code': '+7', 'code': 'KZ'},
   {'name': 'Kenya', 'dial_code': '+254', 'code': 'KE'},
   {'name': 'Kiribati', 'dial_code': '+686', 'code': 'KI'},
   {
-    'name': 'Korea, Democratic Peoples Republic of Korea',
+    'name': 'Korea (Democratic People\s Republic of)',
```
Comment (dnys1) [inline packages/amplify_authenticator/tool/countries.dart:126]: Typo?

File: packages/amplify_authenticator/lib/src/l10n/country_resolver.dart (hunk 2 above)
Comment (haverchuck) [reply packages/amplify_authenticator/lib/src/l10n/country_resolver.dart:1]: Hmm sorry, I thought I removed this.

File: packages/amplify_authenticator/lib/src/l10n/country_resolver.dart (hunk 1 above)
Comment (haverchuck) [reply packages/amplify_authenticator/lib/src/l10n/country_resolver.dart:23]: @dnys1 I updated the l10n generate script to generate country localizations, and then run `flutter format` on the generated files. Let me know what you think.

**Review by Jordan-Nelson — APPROVED** [review]

**Review by dnys1 — APPROVED** [review]
Looks good, thanks 👍
